tests/integration/components/select/state-select-test.js

```diff
@@ -0,0 +1,31 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import Ember from 'ember';
+import hbs from 'htmlbars-inline-precompile';
+import selectState from '../../../pages/components/select/state-select';
+import PageObject from 'ember-cli-page-object';
+
+const {
+  get,
+  set
+} = Ember;
+
+let page = PageObject.create(selectState);
+
+moduleForComponent('select/state-select', 'Integration | Component | select/state select', {
+  integration: true,
+  beforeEach() {
+    page.setContext(this);
+  },
+  afterEach() {
+    page.setContext(this);
```

Comment:
This should be `page.removeContext();`. We may have this wrong in some other tests. Like country, it looks like. Can fix in this PR.